packages/components/inputs/localized-text-input/src/localized-text-input.tsx

```diff
@@ -184,23 +202,26 @@ const LocalizedInput = (props: TLocalizedInputProps) => {
   }
 
   return (
-    <div
-      key={props.language}
-      css={css`
-        width: 100%;
-        position: relative;
-        display: flex;
-      `}
-    >
-      <label htmlFor={props.id} css={getLanguageLabelStyles(props)}>
-        {props.language.toUpperCase()}
-      </label>
-      <TextInput
-        {...props}
-        onChange={handleChange}
-        css={getLocalizedInputStyles}
-      />
-    </div>
+    <Stack scale="xs">
+      <div
+        key={props.language}
+        css={css`
+          width: 100%;
+          position: relative;
+          display: flex;
+        `}
+      >
+        <label htmlFor={props.id} css={getLanguageLabelStyles(props)}>
+          {props.language.toUpperCase()}
+        </label>
+        <TextInput
+          {...props}
+          onChange={handleChange}
+          css={getLocalizedInputStyles}
+        />
+      </div>
+      {props.warning && <div>{props.warning}</div>}
```

Comment:
In order to keep code consistency and readability, I think it would be better to render the warning message beside where we render the error message (down below in line 332).
Doing so, we don't need to change anything in this `LocalizedInput` internal component.